botocore/httpsession.py

```diff
@@ -20,7 +20,9 @@
     OP_NO_SSLv2,
     OP_NO_SSLv3,
     ssl,
+    is_ipaddress,
```

Comment:
What's the minimum version of urllib3 this is available in?

Reply:
Looks like it was added into 1.23 release ([commit](https://github.com/urllib3/urllib3/commit/7c216f433e39e184b84cbfa49e41135a89e4baa0))